tests/ssr.test.tsx

```diff
@@ -19,11 +19,9 @@ const useBearStore = create<BearStoreState & BearStoreAction>((set) => ({
 }))
 
 function Counter() {
-  const { bears, increasePopulation } = useBearStore(
-    ({ bears, increasePopulation }) => ({
-      bears,
-      increasePopulation,
-    }),
```

Comment:
I would say it worked before by chance. This selector creates a new object every time, which isn't supported in v5. We should use a memoized selector, `zutand/traditional` or 👇 .

Reply:
Or, was this test intentional to check the behavior with such selectors? @TkDodo

Reply:
Hmm I think I copied this way if creating a store from somewhere. Definitely not intentional

Reply:
Thanks for confirming. That was the reason I introduced `useMemoSelector`, without looking at the test code. I believe it is an unnecessary hack.